### ISSUE TITLE
Table can't use ul/ol

### ISSUE THREAD

**Issue by webhao** [soccerloway/quill-better-table#68]
![download](https://user-images.githubusercontent.com/23170460/99641563-634deb80-2a85-11eb-8c7a-e5741dc8ee34.png)

**Comment by WongYAQi** [comment]
``` javascript
// in table.js
const List = Quill.import('formats/list')
const Block = Quill.import('blots/block')
...
TableCell.allowedChildren = [..., List.requiredContainer, Block]
```

`List.requiredContainer` can make you add ul, and `Block` can preserve the TableCell after cancel ul format

**Comment by weiQuill** [comment]
@WongYAQi It's not enough for adding ul/ol in table cell.
To achieve it, the `listItem` blot and `listContainer` blot need to be modified accordingly (like table cell line). Currently there is no support for ul/ol in table cell.

**Comment by WongYAQi** [comment]
> @WongYAQi It's not enough for adding ul/ol in table cell.
> To achieve it, the `listItem` blot and `listContainer` blot need to be modified accordingly (like table cell line). Currently there is no support for ul/ol in table cell.

yes, I found that there was problem when merge two list cell, and `checkMerge` function would throw error.  
So I realized that "add ui in table" would be a long-term job.But this may be helpful when just show the ul element with this little change.

**Comment by dbllm** [comment]
> ```js
> // in table.js
> const List = Quill.import('formats/list')
> const Block = Quill.import('blots/block')
> ...
> TableCell.allowedChildren = [..., List.requiredContainer, Block]
> ```
> 
> `List.requiredContainer` can make you add ul, and `Block` can preserve the TableCell after cancel ul format

I'll do as you say, It's half done, The other half is the problem of setting, Didn't you solve it?

**Comment by dbllm** [comment]
The `setContents` will go wrong
